src/tests.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+import sys
+import os
+from pdbvangogh import api
+import tensorflow as tf
+import random
+import numpy as np
+from data_models import *
+
+np.random.seed(42)
+tf.random.set_seed(42)
+random.seed(42)
+
+def test_test_func():
+    assert(True)
+    assert(api.test_func())
+def test_pdbvangogh():
+    api.pdbvangogh(background_image = 'tests/in/pdx.png', 
+                   content_image = 'tests/in/2l1v.png', 
+                   style_image = 'tests/in/starry_night.png',
+                   save_prefix = 'tests/out/2l1v_pdx_starry_night',
+                   content_size=100,
+                   background_size = 200,
+                   background_hyperparameters=vgg19_transfer_parameters(style_weight=1e-2, epochs=1, steps_per_epoch=50),
+                   content_hyperparameters=vgg19_transfer_parameters(style_weight=1e-4, epochs=1, steps_per_epoch=50)
+                   )
+    api.pdbvangogh(background_image = 'tests/in/pdx.png', 
+                   pdb_id = '2HYY',
+                   out_dir = 'tests/out/',
+                   style_image = 'tests/in/starry_night.png',
+                   save_prefix = 'tests/out/2HYY_pdx_starry_night',
+                   content_size=100,
+                   background_size = 200,
+                   background_hyperparameters=vgg19_transfer_parameters(style_weight=1e-2, epochs=1, steps_per_epoch=50),
+                   content_hyperparameters=vgg19_transfer_parameters(style_weight=1e-4, epochs=1, steps_per_epoch=50)
+                   )
+    assert(True)
```

Comment:
Nice, some tests :) I'm guessing you just do a visual or manual diff inspection for now? Though may be worth just adding a couple more basic asserts for now that like the image is output to the path that you expect, etc. Or you can even use something like [imgdiff](https://pypi.org/project/imgdiff/#:~:text=A%20command%2Dline%20tool%20that,a%20version%20control%20tool%2C%20e.g.) if you want to make it entirely procedural 

Another fun thing you can add later is to use github actions to automatically run your tests e.g. on pull request, etc.